schema.sql

```diff
@@ -51,3 +51,18 @@ vets INT REFERENCES vets(id),
 visit_date DATE,
 PRIMARY KEY (animals, vets)
 );
+
+DROP TABLE visits;
+
+CREATE TABLE IF NOT EXISTS visits(
+  animals INT REFERENCES animals(id),
+  vets INT REFERENCES vets(id),
+  visit_date TIMESTAMP NOT NULL,
+  PRIMARY KEY (animals, vets, visit_date)
+);
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX idx_visits_animal_id ON visits(animals);
+CREATE INDEX idx_visits_vet_id_animal_id ON visits(vets, animals);
```

Comment:
Great job team 🤝

- [ ] As I can clearly see that you had some fun with the `explain analyze` and I am sure that in the future you will know how to start the investigation for some slow DB queries, it's time to approve this project :)